package.json

```diff
@@ -7,7 +7,12 @@
     "react-component-variations": "./bin.js"
   },
   "scripts": {
-    "lint": "eslint .",
+    "prebuild": "npm run clean",
+    "build": "babel src -d .",
+    "build:watch": "npm run build -- --watch",
+    "clean": "rm forEachVariation.js",
```

Comment:
this should also `rimraf build`

Reply:
I'm not actually building to a `build` directory because it will break our current import structure, right?

Reply:
oh duh. lol yes sorry, you're correct. What about `bin.js`?

Reply:
I chose to just transpile `forEachVariation` at this point. Will defer to you if you prefer to transpile it all, including `bin.js` and/or `validate.js`